autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -243,6 +262,20 @@ def fit(train_data,
                     Note: a list of hyper-parameters dicts can be passed; each set will create different version of the model.
                     Note: Hyperparameter tuning is disabled for this model.
                     Note: 'penalty' parameter can be used for regression to specify regularization method: 'L1' and 'L2' values are supported.
+                Advanced functionality: Custom AutoGluon model parameters
+                    These parameters are optional and can be specified in any model's hyperparameters.
+                        Example: `hyperparameters = {'RF': {..., '_ag_args': {'name_suffix': 'CustomModelSuffix', 'disable_in_hpo': True}}`
+                    _ag_args: Dictionary of customization options related to AutoGluon.
+                        Valid keys:
+                            name: (str) The name of the model. This overrides AutoGluon's naming logic and all other name arguments if present.
+                            name_main: (str) The main name of the model. In 'RandomForestClassifier', this is 'RandomForest'.
+                            name_prefix: (str) Add a custom prefix to the model name. Unused by default.
+                            name_type_suffix: (str) Override the type suffix of the model name. In 'RandomForestClassifier', this is 'Classifier'. This comes before 'name_suffix'.
+                            name_suffix: (str) Add a custom suffix to the model name. Unused by default.
+                            priority: (int) Determines the order in which the model is trained. Larger values result in the model being trained earlier. Default values range from 100 (RF) to 0 (custom), dictated by model type.
```

Comment:
Add: "If you want this model to be trained first, set priority = X"

Reply:
Added